tests/test_repo_crawler.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+import requests
+from orgwarden.repo_crawler import Repository, fetch_org_repos
+
+SSEC_ORG_NAME = "gt-sse-center"
```

Comment:
Let's not test against SSEC repositories. While `gt-sse-center` will use this functionality, this repository shouldn't necessarily know that. However, this repo resides in the `gt-tech-ai` organization, which means that it is more natural for this repo to have knowledge about its sibling repositories.